Classes/GlobalStateExplorers/DatabaseBrowser/FLEXDatabaseManager.h

```diff
@@ -13,10 +13,10 @@
 
 #import <Foundation/Foundation.h>
 
-@interface FLEXDatabaseManager : NSObject
+@protocol FLEXDatabaseManager <NSObject>
 
-
-- (instancetype)initWithPath:(NSString*)aPath;
+@optional
```

Comment:
:thumbsup: for making this a protocol rather than subclassing.

Could we make these `required`? Looks like we're not doing `respondsToSelector:` check anyways, and I'd prefer the added compiler safety that comes with `required`.